OWML.ModLoader/ModData.cs

```diff
@@ -77,17 +79,15 @@ private void MakeConfigConsistentWithDefault()
             AddMissingDefaults(DefaultConfig);
         }
 
-        private bool UpdateSelector(string key, object userSetting, JObject modderSetting)
+        private void UpdateSelector(string key, object userSetting, JObject modSetting)
         {
-            var options = modderSetting["options"].ToObject<List<string>>();
+            var options = modSetting["options"].ToObject<List<string>>();
             var userString = userSetting is JObject objectValue ? (string)objectValue["value"] : Convert.ToString(userSetting);
-            Config.Settings[key] = modderSetting;
-            var isInOptions = options.Contains(userString);
-            if (isInOptions)
+            Config.Settings[key] = modSetting;
+            if (options.Contains(userString))
             {
                 Config.SetSettingsValue(key, userString);
             }
-            return isInOptions;
```

Comment:
nope, that's actually important (wait for new PR a bit)

Reply:
reverted